postman/Create and Schedule Boadcast.postman_collection.json

```diff
@@ -0,0 +1,506 @@
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "https://api.aweber.com/1.0/accounts/{{accountId}}/lists/{{listId}}/broadcasts?status=sent",
+					"protocol": "https",
+					"host": [
+						"api",
+						"aweber",
+						"com"
+					],
+					"path": [
+						"1.0",
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"broadcasts"
+					],
+					"query": [
+						{
+							"key": "status",
+							"value": "sent"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Create a broadcast",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "ade9b194-0437-4033-93d6-5b49b98671a0",
+						"exec": [
+							"pm.test(\"Successfully created a new broadcast\", function(){",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"pm.test(\"Get broadcast self link\", function() {",
+							"    var jsonData = pm.response.json();",
+							"    pm.globals.set(\"broadcast_self_link\",jsonData.self_link )",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"name": "Content-Type",
+						"value": "application/x-www-form-urlencoded",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "urlencoded",
+					"urlencoded": [
+						{
+							"key": "body_html",
+							"value": "{{body_html}}",
+							"type": "text"
+						},
+						{
+							"key": "body_text",
+							"value": "{{body_text}}",
+							"type": "text"
+						},
+						{
+							"key": "click_tracking_enabled",
+							"value": "{{click_tracking_enabled}}",
+							"type": "text"
+						},
+						{
+							"key": "exclude_lists",
+							"value": "{{exclude_lists}}",
+							"type": "text"
+						},
+						{
+							"key": "facebook_integration",
+							"value": "{{facebook_integration}}",
+							"type": "text"
+						},
+						{
+							"key": "include_lists",
+							"value": "{{include_lists}}",
+							"type": "text"
+						},
+						{
+							"key": "is_archived",
+							"value": "{{is_archived}}",
+							"type": "text"
+						},
+						{
+							"key": "notify_on_send",
+							"value": "{{notify_on_send}}",
+							"type": "text"
+						},
+						{
+							"key": "subject",
+							"value": "{{subject}}",
+							"type": "text"
+						},
+						{
+							"key": "twitter_integration",
+							"value": "{{twitter_integration}}",
+							"type": "text"
+						}
+					]
+				},
+				"url": {
+					"raw": "https://api.aweber.com/1.0/accounts/{{accountId}}/lists/{{listId}}/broadcasts",
+					"protocol": "https",
+					"host": [
+						"api",
+						"aweber",
+						"com"
+					],
+					"path": [
+						"1.0",
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"broadcasts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Schedule the newly created broadcast",
```

Comment:
I'm not sure how it's possible for scheduling to succeed when creating a broadcast fails. Is it holding onto a previous broadcast link? 
![screen shot 2018-11-21 at 1 44 38 pm](https://user-images.githubusercontent.com/4196901/48862214-a50f0300-ed93-11e8-8f98-92b0dce2dc24.png)